release/readme.html

```diff
@@ -110,40 +128,71 @@ <h5 class="card-title">Yes... and no.</h5>
+                        <li class="list-group-item transition-object-name">Boiler:Steam</li>
+                        <li class="list-group-item transition-object-name">Chiller:EngineDriven</li>
+                        <li class="list-group-item transition-object-name">Chiller:CombustionTurbine</li>
+                        <li class="list-group-item transition-object-name">ChillerHeater:Absorption:DirectFired</li>
+                        <li class="list-group-item transition-object-name">Coil:Cooling:DX:MultiSpeed</li>
+                        <li class="list-group-item transition-object-name">Coil:Heating:Fuel</li>
+                        <li class="list-group-item transition-object-name">Coil:Heating:DX:MultiSpeed</li>
+                        <li class="list-group-item transition-object-name">EnergyManagementSystem:Program</li>
+                        <li class="list-group-item transition-object-name">EnergyManagementSystem:Subroutine</li>
+                        <li class="list-group-item transition-object-name">EnergyManagementSystem:MeteredOutputVariable</li>
+                        <li class="list-group-item transition-object-name">Exterior:FuelEquipment</li>
+                        <li class="list-group-item transition-object-name">FuelFactors</li>
+                        <li class="list-group-item transition-object-name">Generator:CombustionTurbine</li>
+                        <li class="list-group-item transition-object-name">Generator:InternalCombustionEngine</li>
+                        <li class="list-group-item transition-object-name">Generator:MicroTurbine</li>
+                        <li class="list-group-item transition-object-name">GlobalGeometryRules</li>
+                        <li class="list-group-item transition-object-name">HeatPump:WaterToWater:EIR:Heating</li>
+                        <li class="list-group-item transition-object-name">HeatPump:WaterToWater:EIR:Cooling</li>
+                        <li class="list-group-item transition-object-name">HVACTemplate:System:VRF</li>
+                        <li class="list-group-item transition-object-name">HVACTemplate:Plant:Boiler</li>
+                        <li class="list-group-item transition-object-name">LifeCycleCost:UsePriceEscalation</li>
+                        <li class="list-group-item transition-object-name">LifeCycleCost:UseAdjustment</li>
+                        <li class="list-group-item transition-object-name">OtherEquipment</li>
+                        <li class="list-group-item transition-object-name">Output:Table:SummaryReports</li>
+                        <li class="list-group-item transition-object-name">ShadowCalculation</li>
+                        <li class="list-group-item transition-object-name">WaterHeater:Mixed</li>
+                        <li class="list-group-item transition-object-name">WaterHeater:Stratified</li>
+                        <li class="list-group-item transition-object-name">ZoneHVAC:HybridUnitaryHVAC</li>
```

Comment:
Some of these show up in the transition utility, because they need to be modified when transitioning other objects, but the objects themselves didn't actually change for this release. Delete these: ZoneHVAC:EquipmentList, AirLoopHVAC:ZoneSplitter, AirLoopHVAC:SupplyPlenum, AirLoopHVAC, RoomAir:Node:AirflowNetwork:HVACEquipment (ugh - this never got mentioned in the transition rules doc), and AirflowNetwork:Distribution:Node, AirflowNetwork:Distribution:Linkage.

Reply:
Thanks, removed them locally!